cmd/thanos/tools.go

```diff
@@ -99,31 +151,31 @@ func checkRules(logger log.Logger, filename string) (int, errors.MultiError) {
 	return numRules, checkErrors
 }
 
-func thanosRuleGroupsToPromRuleGroups(ruleGroups ThanosRuleGroups) rulefmt.RuleGroups {
-	promRuleGroups := rulefmt.RuleGroups{Groups: []rulefmt.RuleGroup{}}
-	for _, g := range ruleGroups.Groups {
-		group := rulefmt.RuleGroup{
-			Name:     g.Name,
-			Interval: g.Interval,
-			Rules:    []rulefmt.Rule{},
-		}
-		for _, r := range g.Rules {
-			group.Rules = append(
-				group.Rules,
-				rulefmt.Rule{
-					Record:      r.Record,
-					Alert:       r.Alert,
-					Expr:        r.Expr,
-					For:         r.For,
-					Labels:      r.Labels,
-					Annotations: r.Annotations,
-				},
-			)
-		}
-		promRuleGroups.Groups = append(
-			promRuleGroups.Groups,
-			group,
-		)
-	}
-	return promRuleGroups
-}
+//func thanosRuleGroupsToPromRuleGroups(ruleGroups ThanosRuleGroups) rulefmt.RuleGroups {
```

Comment:
What's this?

Reply:
Because I copied `Validate()` method temporarily, this method was unused. I kept it in comment because once we switch back to using upstream `Validate()`, we will need it again. Do you want me to remove it altogether?

Reply:
Well git is versioning system you know... we don't need another backuping with comments (:

Reply:
Makes sense XD